message_definitions/v1.0/development.xml

```diff
@@ -38,5 +44,25 @@
       <field type="float" name="raw_press" units="hPa">Raw differential pressure. NaN for value unknown/not supplied.</field>
       <field type="uint8_t" name="flags" enum="AIRSPEED_SENSOR_FLAGS">Airspeed sensor flags.</field>
     </message>
+    <message id="420" name="RADIO_RC_CHANNELS">
+      <description>RC channel inputs for a flight controller and associated components; the message does not replace the RC_CHANNELS and similar output messages.
```

Comment:
See https://github.com/ArduPilot/mavlink/pull/343#issuecomment-1918417134 > This is not sufficient for "most people" to understand that this is the output of an Rx receiver, replacing other protocols.